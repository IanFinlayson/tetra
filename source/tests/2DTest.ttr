def main():
   x = [[1,2],[3,4]]
   print("Just created x and x = ", x, "\n")
   x[1][1] = 9999
   print("Just set x[1][1] = 9999 and x = ", x, "\n")
   z = 0
   print("Just created z and z = ", z, "\n")
   for i in x:
      print("i = ", i, "\n")
      if z == 0:
         print("z = 0, see: ", z, "\n")
         d = i[0]
         print("Just set d = i[0] and d = ", d, "\n")
         k = i[1]
         print("Just set k = i[1] and k = ", k, "\n")
      else:
         print("z != 0, see: ", z, "\n")
         p = i[0]
         print("Just set p = i[0] and p = ", p, "\n")
         c = i[1]
         print("Just set c = i[1] and c = ", c, "\n")
      z = z + 1
      print("Just added 1 to z and z = ", z, "\n")
   print("x[0][1] = ", x[0][1],"\n")
