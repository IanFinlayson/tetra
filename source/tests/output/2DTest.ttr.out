Just created x and x = [[1, 2], [3, 4]]
Just set x[1][1] = 9999 and x = [[1, 2], [3, 9999]]
Just created z and z = 0
i = [1, 2]
z = 0, see: 0
Just set d = i[0] and d = 1
Just set k = i[1] and k = 2
Just added 1 to z and z = 1
i = [3, 9999]
z != 0, see: 1
Just set p = i[0] and p = 3
Just set c = i[1] and c = 9999
Just added 1 to z and z = 2
x[0][1] = 2

